pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -242,7 +245,70 @@ function subvolume_delete(volume, subvol, mount_point_in_parent, card) {
     });
 }
 
-export function make_btrfs_subvolume_page(parent, volume, subvol) {
+function dirname(path) {
+    const i = path.lastIndexOf("/");
+    if (i < 0)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20067-20240301-083916-612a49d9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)